src/auth/models.hpp

```diff
@@ -62,10 +62,10 @@ constexpr inline uint64_t operator&(uint64_t lhs, LabelPermission rhs) {
   return (lhs & static_cast<uint64_t>(rhs)) != 0;
 }
 
-constexpr uint64_t LabelPermissionAll = memgraph::auth::LabelPermission::CREATE_DELETE |
-                                        memgraph::auth::LabelPermission::EDIT | memgraph::auth::LabelPermission::READ;
-constexpr uint64_t LabelPermissionMax = static_cast<uint64_t>(memgraph::auth::LabelPermission::CREATE_DELETE);
-constexpr uint64_t LabelPermissionMin = static_cast<uint64_t>(memgraph::auth::LabelPermission::READ);
+constexpr uint64_t kLabelPermissionAll = memgraph::auth::LabelPermission::CREATE_DELETE |
+                                         memgraph::auth::LabelPermission::EDIT | memgraph::auth::LabelPermission::READ;
```

Comment:
I think `EDIT` is confusing because we never use that word, `SET` clause in Cypher is considered to be an **update** clause, so I think `UPDATE` is a better word here 😄